employee-rostering-frontend/src/ui/pages/shift/ShiftRosterPage.test.tsx

```diff
@@ -310,6 +310,23 @@ describe('Shift Roster Page', () => {
       pinnedByUser: false
     });
   });
+  
+  it('should call updateShift when an event is updated', () => {
+    const shiftRosterPage = shallow(<ShiftRosterPage
+      {...baseProps}
+    />);
+    const newDateStart = moment(startDate).add(7, "days").toDate();
+    const newDateEnd = moment(endDate).add(7, "days").toDate();
+    shiftRosterPage.find('Schedule').simulate("updateEvent", shift, newDateStart,
```

Comment:
Nitpick: can use `find(Schedule)` (an imported component type instead of a string) => smoother refactoring and discoverable usage.